start-stop-lb-services.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+usage()
+{
+cat <<EOF
+Usage: $0
+    --stop <Stop LightBeam Cluster Services>
+    --start <Start LightBeam Cluster Services>
+    --lb-namespace <Lightbeam cluster namespace>
+
+Examples:
+    # Stop services.
+    ./start-stop-lb-services.sh --stop
+
+    # Start services.
+    ./start-stop-lb-services.sh --start
+
+EOF
+exit 1
+}
+
+while [ "$1" != "" ]; do
+    case $1 in
+    --stop)             STOP_SERVICES="True"
+                          ;;
+    --start)            START_SERVICES="True"
+                          ;;
+    --lb-namespace)     shift
+                        NAMESPACE=$1
+                          ;;
+    -h | --help )   usage
+                    ;;
+    * )             usage
+    esac
+    shift
+done
+
+LB_DEFAULT_NAMESPACE="lightbeam"
+API_GATEWAY_DEPLOYMENT_NAME="lightbeam-api-gateway"
+PRESIDIO_SERVER_DEPLOYMENT_NAME="lightbeam-presidio-server"
+TEXT_EXTRACTION_DEPLOYMENT_NAME="lightbeam-text-extraction"
+TIKA_DEPLOYMENT_NAME="lightbeam-tika-server"
+DATASOURCE_STATS_AGGREGATOR_DEPLOY_NAME="lightbeam-datasource-stats-aggregator"
+POLICY_ENGINE_DEPLOY_NAME="lightbeam-policy-engine"
+POLICY_CONSUMER_DEPLOY_NAME="lightbeam-policy-consumer"
+DATASOURCE_STATS_AGGREGATOR_KAFKA_QUEUE="datasource-stats"
+
+if [ -z ${NAMESPACE} ]; then
+  NAMESPACE=$LB_DEFAULT_NAMESPACE
+fi
+
+if [[ -z ${STOP_SERVICES} || -z ${START_SERVICES} ]]; then
+   usage
+fi
+
+if [ ${STOP_SERVICES} ]; then
+   CRONJOB_SUSPEND_ENABLED="true"
+   REPLICAS="0"
+fi
+
+if [ ${START_SERVICES} ]; then
+   CRONJOB_SUSPEND_ENABLED="false"
+   REPLICAS="1"
```

Comment:
How do you know it should be 1? The ideal fix is to save the number of replicas when stop is called to a file so that start will consult this file instead. That way, we can start supporting replica counts of greater than 1 as well. It will also ensure that if some service is not enabled, we will not end up restarting it.